Anchal_Hora/panel/WebApplicationPanel/DbModels/panelContext.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace WebApplicationPanel.DbModels
+{
+    public partial class panelContext : DbContext
+    {
+        public panelContext()
+        {
+        }
+
+        public panelContext(DbContextOptions<panelContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<EmpInformation> EmpInformation { get; set; }
+        public object Panel { get; internal set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer("server=CYG366\\SQLEXPRESS; database=panel; Trusted_Connection=True;");
```

Comment:
Why is the Connection String not in AppSettings.json

Reply:
Added Connection String in AppSettings.json